README.md

```diff
@@ -1,11 +1,22 @@
 PMS_Updater
 ===========
+Note: I think python temporarily broke in 1.16.6.1559, if you happen to be on this release, or any other that broke python, you can manually update to get around something like this.
+
+To get around it download the file manually and install like this,
+
+cd /tmp
+
+fetch https://downloads.plex.tv/plex-media-server-new/1.16.6.1592-b9d49bdb7/freebsd/PlexMediaServer-1.16.6.1592-b9d49bdb7-FreeBSD-amd64.tar.bz2
+
+/root/PMS_Updater.sh -l /tmp/PlexMediaServer-1.16.6.1592-b9d49bdb7-FreeBSD-amd64.tar.bz2
+
+==========
```

Comment:
What about moving this in a new Troubleshooting section at the end of the readme?

````md
Troubleshooting
============

Python temporarily broke in some Plex version, probably in 1.16.6.1559.
If you happen to be on this release, or any other that broke python, you
can manually update to get around something like this:

```sh
cd /tmp
fetch https://downloads.plex.tv/plex-media-server-new/1.16.6.1592-b9d49bdb7/freebsd/PlexMediaServer-1.16.6.1592-b9d49bdb7-FreeBSD-amd64.tar.bz2
/root/PMS_Updater.sh -l /tmp/PlexMediaServer-1.16.6.1592-b9d49bdb7-FreeBSD-amd64.tar.bz2
```
````